### DISCUSSION TITLE
Second Gnome Edition in the style of Zorin OS 16

### DISCUSSION THREAD

**Discussion by JohnnyCalavera** [SpiralLinux/SpiralLinux-project#30] [General]
Hello again,

we support a lot of users that are used to windows desktops and we're happy to see that most DE configs in SpiralLinux come very close to what they expect to see. There are use cases for the touch-friendly gnome way of doing things, but we'd like to ask if you are willing to provide a second iso for gnome, Sam. We know that there's a "Builder", but we lack the knowledge to create what we are looking for.
Would you be interested in providing a second SpiralLinux gnome config that looks like the gnome desktop of Zorin OS 16 (core)? It calls itself "windows 10 inspired" and reminds of the standard cinnamon look and feel in a lot of places.

Please let us know what you think.

**Comment by geckolinux** [comment]
Hi Johnny, thanks for this interesting idea. I wouldn't be opposed to the general idea, here are a few more questions and considerations:

- What exactly about the Zorin OS core desktop layout makes it desirable? (I haven't used it, only read reviews.)
- If Zorin uses custom Gnome Shell extensions to achieve its layout then it won't be possible to implement that in SpiralLinux, since SpiralLinux only ships software that can be reliably upgraded from upstream sources.
- Have you considered the Dash to Panel extension (included but not enabled by default in SpiralLinux Gnome) to replicate the Windows / Zorin layout you prefer?
- Have you considered the standard Cinnamon layout? In my experience with Windows switchers, Cinnamon is by far the easiest for them. You could use the other task bar option of the Windows-style squares for the active and pinned windows.
- In general I have found that trying to imitate the Windows look and feel does not really help new users, because it makes them think that the OS will work just like Windows, when in reality it's more of a _"fachada"_ covering the very different Linux system behind it. I do think it's important to include *features* from Windows in Linux distros (i.e. an application menu in the bottom left corner, a taskbar, a system tray, etc.) but without  necessarily making those features aesthetically similar to Windows.
- It's worth considering that instead of creating more editions or spins, it's also possible in the case of a Gnome / Cinnamon / Mate / Budgie system to simply replace the `~/.config/dconf/user` file with another one to completely change the configuration to something else.

Hope this helps!

**Reply by JohnnyCalavera** [reply]
In short: we like the idea of having gnome apps / tools (like nautilus and so on), but the look and feel that you would get from cinnamon, too.
As you mentioned it's about things like _application menu in the bottom left corner, a taskbar, a system tray_ and not about copying windows in any other way.
We used Dash to Panel in 2019 for our pure Debian installations and it did it's job well. SpiralLinux Cinnamon is about to become our default for now which means that we don't necessarily have to create a solution with gnome, but we wanted to ask at least ^^ 
I wasn't able to determine how Zorin OS's tweaks are done. Of course gnome shell extensions or packages from Zorin/Ubuntu would be a problem. Tomorrow I will try to export the config of Zorin OS 16 to see if it is easily transferable - thanks for your thoughts.

PS: Here is an official screenshot of Zorin's gnome desktop (no running applications unfortunately)
![zorin](https://user-images.githubusercontent.com/16836649/176301499-7a71674e-cf26-4deb-9a5d-96fb692a7aa5.jpg)

**Reply by geckolinux** [reply]
I see, thanks. I'm very happy to know that SpiralLinux is useful for your deployments.
I would encourage you to look at the SpiralLinux Cinnamon spin for that, it would be very easy to replicate that look and feel with Cinnamon.

**Reply by methanoid** [reply]
> Hi Johnny, thanks for this interesting idea. I wouldn't be opposed to the general idea, here are a few more questions and considerations:
> 
> * Have you considered the Dash to Panel extension (included but not enabled by default in SpiralLinux Gnome) to replicate the Windows / Zorin layout you prefer?

Hi

Was just about to post a thread but searched first ;-) 

Is the lack of Dash to Panel being enabled by default why Spiral Gnome doesnt give a stock Gnome look? I've always preferred Plasma desktop but wanted to try Gnome alongside dual booting to see which I like most, so was installing Spiral Gnome today

**Reply by geckolinux** [reply]
Hi there @methanoid , the SpiralLinux Gnome edition is intended to replicate the Ubuntu Gnome look. For a Windows / Zorin bottom panel I would recommend the Cinnamon edition.

**Comment by methanoid** [comment]
I was looking for the Ubuntu Gnome look.. but since I had upgraded to Bookworm and Gnome 43 it didnt look anything like that. A bit of playing with the settings seemed to have fixed it (not rounded corners on "dock" ??)
